src/graph/viewport.ts

```diff
@@ -101,7 +101,7 @@ export class Viewport {
         $sourceGroup.classList.add('hovered');
         $prevHovered = $sourceGroup;
         let $edge = edgeFrom($sourceGroup.id);
-        $edge.classList.add('hovered');
+        $edge && $edge.classList.add('hovered');
```

Comment:
@taras Thanks for PR 👍 
But it's workaround not a fix.
If you do similar things like selecting node or edge you should get similar error.
So can you please create a minimal reproducible example?
Also can you please try replacing all instances `document` with `window.self.document` in this file?

Reply:
> But it's workaround not a fix.
True. 

> So can you please create a minimal reproducible example?

Yes, not today but I will in the near future.

> Also can you please try replacing all instances document with window.self.document in this file?

I'll give it a try and let you know.